lib/db/mysql/schema.sql

```diff
@@ -46,7 +46,8 @@ CREATE TABLE IF NOT EXISTS tokens (
   type VARCHAR(16) NOT NULL,
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  expiresAt TIMESTAMP NOT NULL
+  expiresAt TIMESTAMP NOT NULL,
+  INDEX idx_expiresAt(expiresAt),
```

Comment:
> Did you want to have CREATE PROCEDURE PURGE_EXPIRED_TOKENS statement here

Yes, the entire create statement; ideally you could run this file and put your database in the correct "current" state without worrying about migrations etc.  It's an unfortunate bit of duplication but having the ability to see the whole schema in a single file is a real benefit.